apiserver/params/internal.go

```diff
@@ -270,8 +281,9 @@ type SetUnitStateArgs struct {
 // SetUnitStateArg holds a unit tag and a map with KV-pairs that represent a local
 // charm state to be persisted by the controller
 type SetUnitStateArg struct {
-	Tag   string            `json:"tag"`
-	State map[string]string `json:"state"`
+	Tag         string            `json:"tag"`
+	State       map[string]string `json:"state"`
```

Comment:
Since we are changing the payload format, can we add an `omitempty` for this field as well?